### PR TITLE
Issue template for support

### PR DESCRIPTION
**Pull request by guperrot**

### CODE REVIEW

**Comment by dhei** [comment]
@guperrot @elamalani I suggest we also ask for information about dev environment, expected behavior, actual behavior and repro steps in the Issue Template. A good example for such template is FB's react native [ISSUE_TEMPLATE](https://github.com/facebook/react-native/blob/master/.github/ISSUE_TEMPLATE.md).

**Comment by ajwhite** [comment]
Recommend guiding users to run `react-native info` for a standard format that is easily obtainable

Eg output:

```
Environment:
  OS: macOS High Sierra 10.13.2
  Node: 6.10.2
  Yarn: 0.24.5
  npm: 3.10.10
  Watchman: 4.7.0
  Xcode: Xcode 9.2 Build version 9C40b
  Android Studio: 3.0 AI-171.4443003

Packages: (wanted => installed)
  react: 16.2.0 => 16.2.0
  react-native: 0.53.3 => 0.53.3
```

**Comment by guperrot** [comment]
Applied suggestions but since not issues are bugs I avoided imposing useless headers, everything is now in markdown comment so that the instructions will not show up in the actual ticket.

**Comment by ajwhite** [comment]
> since not all issues are bugs

Oh btw, @guperrot -- have you seen Github's new multiple issue template feature?
https://blog.github.com/2018-01-25-multiple-issue-and-pull-request-templates/

May be useful for your concern. We share the same hesitation in a few of our OSS projects, but this has made it much easier to manage now

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  (for example a build issue and you are not using our SDK, or your issue is about CodePush),
+  then please create the ticket on Intercom instead (using the blue chat button on any https://appcenter.ms page).
+
+  Please attach some technical information about your environment,
+  run the following 2 commands and attach output to the ticket:
+
+  `react-native info`
+  `pod --version`
+
+  For bugs please describe the reproduction scenario and expected behavior.
```
Comment (elamalani) [inline ISSUE_TEMPLATE.md]: a comma "," after bugs

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!--
+  Thanks for your interest in using the App Center SDK for React Native.
+
+  If your issue is not about the Analytics/Crashes/Push SDK for React Native,
+  (for example a build issue and you are not using our SDK, or your issue is about CodePush),
+  then please create the ticket on Intercom instead (using the blue chat button on any https://appcenter.ms page).
+
+  Please attach some technical information about your environment,
+  run the following 2 commands and attach output to the ticket:
```
Comment (elamalani) [inline ISSUE_TEMPLATE.md]: attach output to the ticket so that we can investigate the issue
/

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<!--
+  Thanks for your interest in using the App Center SDK for React Native.
+
+  If your issue is not about the Analytics/Crashes/Push SDK for React Native,
+  (for example a build issue and you are not using our SDK, or your issue is about CodePush),
```
Comment (elamalani) [inline ISSUE_TEMPLATE.md]: May be we can phrase it like this -

If you have an issue in React Native platform but **have not** integrated App Center SDK in your application, please contact support by clicking on the blue icon on the lower right of App Center portal (https://appcenter.ms). Our support team will get in touch to assist you.

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (elamalani) [reply ISSUE_TEMPLATE.md]: We can rephrase it like this -

Any issues related to SDK will require you to attach some technical information about your environment. So please run the following two commands and attach the output to your ticket:

**Comment by elamalani** [comment]
@guperrot We should also save it as a canned message on Intercom and let our CSS team know that they should add this message if there is any RN SDK ticket open with this missing information.

**Pull request merged**